internal-workflows/pr-fixer/README.md

```
@@ -0,0 +1,59 @@
+# 1. Fetch all data for the PR
+./scripts/fetch-pr.sh --repo owner/repo --pr 123
+
+# 2. Run the agent to fix the PR
+#    (via Ambient Code Platform or Claude Code)
+```
+
+The agent reads the fetched data, checks out the PR branch, and works through: rebase, reviewer feedback, lints, tests, and push.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)